Report/CostBreakdownChart.php

```diff
@@ -70,6 +70,9 @@ public function __construct(CostBreakdownReporter $reporter)
     public function getChart($campaignId, $dateFrom, $dateTo)
     {
         $report = $this->reporter->getReport($campaignId, $dateFrom, $dateTo);
+        if (empty($report)) {
+            return $report;
+        }
```

Comment:
Is this safe? I am not sure what the caller of getChart is expecting, but this could very well be an empty array or null. Are both values acceptable?

Reply:
Good catch, I just tested if returning null causes any errors and didn't have a problem.
I'll add a check for null and return an empty array just to be safe.